inductiva/api/methods.py

```diff
@@ -428,36 +444,20 @@ def run_simulation(
     resource_pool_id: Optional[UUID] = None,
 ) -> pathlib.Path:
     """Run a simulation synchronously via Inductiva Web API."""
+    task = run_async_simulation(api_method_name, input_dir, params,
```

Comment:
This method then should be called differently, if it can be used inside the run_simulation.... However, note my comment below and we will adapt it on the next iteration.

Reply:
I don't think there's any issue with the name. It's natural that "run_simulation" involves running the simulation asynchronously and then blocking waiting for it to end. This way there's only one path irregardless if the user calls a sync simulation or calls it asynchronously and then waits, the logs and behavior will be the same.